### PR TITLE
Added toggle Render type to interactive builder

### PR DESCRIPTION
**Pull request by Kinotijoan**

### CODE REVIEW

**Review by CynthiaKamau — COMMENTED** [review]
Thanks @Kinotijoan , please do the same for the `edit-question.modal`

**Comment by Kinotijoan** [comment]
> Thanks @Kinotijoan , please do the same for the `edit-question.modal`

Sure, I will

File: src/components/interactive-builder/add-question.modal.tsx [hunk 1]
```diff
@@ -495,6 +503,29 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
...
+                        value={t(toggleLabelTrue || '')}
+                        onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                          setToggleLabelTrue(event.target.value)
+                        }
+                        placeholder={t('On')}
+                        required
+                      />
+                      <TextInput
+                        id="lableFalse"
+                        labelText={t('Label false', 'Lable false')}
```
Comment (NethmiRodrigo) [inline src/components/interactive-builder/add-question.modal.tsx:520]:
```suggestion
                        labelText={t('Label false', 'Label false')}
```

File: src/components/interactive-builder/edit-question.modal.tsx [hunk 2]
```diff
@@ -463,6 +471,25 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
...
+                  id="lableTrue"
+                  labelText={t('Label true', 'Label true')}
+                  value={t(toggleLabelTrue || '')}
+                  onChange={(event: React.ChangeEvent<HTMLInputElement>) => setToggleLabelTrue(event.target.value)}
+                  placeholder={t('On')}
+                  required
+                />
+                <TextInput
+                  id="lableFalse"
+                  labelText={t('Label false', 'Lable false')}
```
Comment (NethmiRodrigo) [inline src/components/interactive-builder/edit-question.modal.tsx:486]:
```suggestion
                  labelText={t('labelFalse', 'Label false')}
```

File: src/components/interactive-builder/edit-question.modal.tsx [hunk 3]
```diff
@@ -463,6 +471,25 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
                 onChange={(event: React.ChangeEvent<HTMLInputElement>) => setRows(event.target.value)}
                 required
               />
+            ) : fieldType === 'toggle' ? (
+              <div>
+                <TextInput
+                  id="lableTrue"
+                  labelText={t('Label true', 'Label true')}
```
Comment (NethmiRodrigo) [inline src/components/interactive-builder/edit-question.modal.tsx:478]:
```suggestion
                  labelText={t('labelTrue', 'Label true')}
```

File: src/components/interactive-builder/edit-question.modal.tsx [hunk 4]
```diff
@@ -463,6 +471,25 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
...
+                  onChange={(event: React.ChangeEvent<HTMLInputElement>) => setToggleLabelTrue(event.target.value)}
+                  placeholder={t('On')}
+                  required
+                />
+                <TextInput
+                  id="lableFalse"
+                  labelText={t('Label false', 'Lable false')}
+                  value={t(toggleLabelFalse || '')}
+                  onChange={(event: React.ChangeEvent<HTMLInputElement>) => setToggleLabelFalse(event.target.value)}
+                  placeholder={t('Off')}
```
Comment (NethmiRodrigo) [inline src/components/interactive-builder/edit-question.modal.tsx:489]:
```suggestion
                  placeholder={t("off", "Off")}
```

File: src/components/interactive-builder/edit-question.modal.tsx [hunk 5]
```diff
@@ -463,6 +471,25 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
...
                 required
               />
+            ) : fieldType === 'toggle' ? (
+              <div>
+                <TextInput
+                  id="lableTrue"
+                  labelText={t('Label true', 'Label true')}
+                  value={t(toggleLabelTrue || '')}
+                  onChange={(event: React.ChangeEvent<HTMLInputElement>) => setToggleLabelTrue(event.target.value)}
+                  placeholder={t('On')}
```
Comment (NethmiRodrigo) [inline src/components/interactive-builder/edit-question.modal.tsx:481]:
```suggestion
                  placeholder={t("on", "On")}
```

**Review by NethmiRodrigo — COMMENTED** [review]
Thanks @Kinotijoan! Just some minor corrections related to translation strings

**Comment by NethmiRodrigo** [comment]
@Kinotijoan could you please update the PR description with the PR template, it helps the reviewers have more context on it 😊

**Pull request closed without merging**